logger/src/lib.rs

```diff
@@ -283,6 +283,7 @@ pub fn install_panic_hook() -> Result<(), Report> {
         .compare_exchange(false, true, Ordering::SeqCst, Ordering::SeqCst)
         .is_ok()
     {
+        println!("Installing eyre hook");
```

Comment:
Why not use `self::prelude::trace!` here? And also maybe a bit more specific:
```suggestion
        println!("Installing eyre hook for test network");
```

Reply:
Actually I meant to remove it.